packages/contracts/test/chugsplash/config.spec.ts

```diff
@@ -0,0 +1,375 @@
+import { expect } from '../setup'
+
+/* Imports: Internal */
+import { parseChugSplashConfig } from '../../src'
+
+describe('ChugSplash config parsing', () => {
+  describe('parseChugSplashConfig', () => {
+    it('should correctly parse a basic config file with no template variables', () => {
+      expect(
+        parseChugSplashConfig({
+          contracts: {
+            MyContract: {
+              address: `0x${'11'.repeat(20)}`,
+              source: 'MyContract',
+              variables: {},
+            },
+          },
+        })
+      ).to.deep.equal({
+        contracts: {
+          MyContract: {
+            address: `0x${'11'.repeat(20)}`,
```

Comment:
And if you leave it as optional chugsplash should CREATE2 calculate what the address will be & fill it in right?

Reply:
Yes, correct. Chugsplash will CREATE2 and fill it in for L1. Just not adding that logic yet since we'll want a whole suite of tests around making sure that's working correctly.